Better clarify option for named parameters in migration guide (#4364)
dart-lang/site-www: src/null-safety/migration-guide.md

Most of the changes that your code needs to be null safe
are easily predictable.
For example, if a variable can be `null`,

> [its type needs a `?` suffix][nullable type]
> and a named parameter that shouldn't be nullable
> needs to be marked [`required`][required]
> or provided a [default value][].

Reviewer: **suggestion:** Run on sentence. Cut into two.
```suggestion
[its type needs a `?` suffix][nullable type]. 
If a named parameter shouldn't be nullable,
mark it [`required`][required]
or give it a [default value][].
```
